test/functional/fixtures/api/es-next/global-hooks/test.js

```diff
@@ -0,0 +1,47 @@
+const path           = require('path');
+const createTestCafe = require('../../../../../../lib');
+
+let testCafe = null;
+
+const runTest = (testName) => {
+    if (!testCafe)
+        throw new Error('"testCafe" isn\'t defined');
+
+    const runner = testCafe.createRunner();
+
+    return runner
+        .filter(test => {
+            return testName ? test === testName : true;
+        })
+        .run();
+};
+
+describe('[API] fixture global before/after hooks', () => {
+    before(async () => {
```

Comment:
Add more tests:
* handle error in global hooks
* check execution order: first global hooks, next - others

Reply:
Tests of handling error are there https://github.com/Aleksey28/testcafe/pull/4/files#diff-3829a93ad38f94eec6c727e1bc1349fc1fd361d5f8ae0bdc0b4c6c433e32c3e8R53

check execution order - completed